libs/analytics/WordPressAnalytics/build.gradle

```diff
@@ -29,21 +11,8 @@ android {
     compileSdkVersion 29
 
     defaultConfig {
-        versionName "1.3.1"
         minSdkVersion 16
         targetSdkVersion 29
     }
 }
 
-publish {
```

Comment:
Was this the name of the old plugin extension for bintray publishing?

Reply:
Yeap: https://github.com/novoda/bintray-release